accepted/00063-hub-iss.md

```diff
@@ -145,35 +158,40 @@ Expected benefits:
 
```

Comment:
Line 150. we don't need to delete anything.

Reply:
Well, in principle we might. What about a channel that used to have a package but now does not have any more? We might want to clean up such a situation.

Reply:
ok that makes sense

Reply:
Same point is being discussed upthread, marking this one resolved